README.md

```diff
@@ -1,64 +1,43 @@
 # Arduino App CLI
 
-`arduino-app-cli` is a command line tool and a service running on Arduino UNO Q boards, that:
+`arduino-app-cli` is a command line tool running on the [Arduino UNO Q](https://docs.arduino.cc/hardware/uno-q/) boards, that manages and runs Arduino Apps (both Linux and microcontroller parts), provides a HTTP daemon mode to expose RestFul APIs, and auto-updates itself and other components.
 
-- manages and runs Arduino Apps on the board (both Linux and microcontroller parts)
-- provides multiple APIs to perform actions and fetch data, used by the front-end (ArduinoAppsLab)
-- auto-updates itself and other components
+[![Test Go status](https://github.com/arduino/arduino-app-cli/actions/workflows/go-test.yml/badge.svg)](https://github.com/arduino/arduino-app-cli/actions/workflows/go-test.yml)
 
-## Environment Variables
+## Docs
 
-The following environment variables are used to configure `arduino-app-cli`:
+For guidance on installation and development, see the [User documentation].
 
-### Application Directories
+## Quickstart
 
-- **`ARDUINO_APP_CLI__APPS_DIR`** Path to the directory where Arduino Apps created by the user are stored.\
-  **Default:** `/home/arduino/ArduinoApps`
+// TODO
```

Comment:
The need to add a quickstart guide should be tracked via an issue instead of a placeholder in the docs.